packages/sdk/src/utils/readPath.ts

```diff
@@ -15,7 +15,7 @@ export default function readPath(src: any, dst: any, ...path: string[]) {
 		field = path.shift();
 		validateJsonFieldName(field);
 		if (path.length) {
-			if (!dst.hasOwnProperty(field)) {
+			if (!Object.prototype.hasOwnProperty.call(dst, field)) {
```

Comment:
Curious, what was the issue here?

Reply:
`hasOwnProperty` could be an assigned property of the object, instead of being the built-in method.